docs/pages/setup/reference/networking.mdx

```diff
@@ -18,22 +22,51 @@ following use cases:
 
 - You have multiple identical services, like proxies, behind a load balancer.
 - You want Teleport to issue an SSH certificate for the service with the additional
-  principals, e.g.host names.
+  principals, e.g., host names.
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+
+Teleport Nodes have an optional `public_addr` property that you can modify in
```

Comment:
Cloud customers likely want to know about the public_addr property for app, db, and desktop access.